app/src/main/java/org/simple/clinic/editpatient/EditPatientScreen.kt

```diff
@@ -315,58 +315,20 @@ class EditPatientScreen(context: Context, attributeSet: AttributeSet) : Relative
   override fun showValidationErrors(errors: Set<EditPatientValidationError>) {
```

Comment:
Rename commit message to `Reformat EditPatientScreen#showValidationErrors`

Reply:
Done